miscellaneous/eggnog_performance.md

```diff
@@ -15,4 +15,34 @@ The results are summarized in the table below:
 | 5               | 1                             | N.A. (43 finished)        | 21 min                       |
 | 10              | 1                             | N.A. (39 finished)        | 42 min                       |
 
-Each node has 3 hrs of wall time.
\ No newline at end of file
+(Each node has 3 hrs of wall time.)
+
+### Test with scratch drive (Recommended by NERSC)
+
+Copied the source files to `$SCRATCH` and ran the same tests. The results are summarized in the table below:
+
+| Number of nodes | Parallelization within a node | Time to finish 50 genomes                         | Average node time per genome |
+|-----------------|-------------------------------|---------------------------------------------------|------------------------------|
+| 1               | 1                             | 54.83 mins                                        | 1.1 min                      |
+| 1               | 2                             | 39.77 mins (both batches)                         | 0.79 min                     |
+| 1               | 5                             | 27.92 mins (first batch), 37.11 mins (last batch) | 0.74 min                     |
```

Comment:
I'm not sure I understand the first vs last batch stuff. Do you mean fastest vs. slowest? IOW for this line you needed 37.11 mins of node time to run 50 genomes in 2 threads of execution?

Reply:
yup. fastest vs slowest. For this line, there are 5 parallelizations with 10 files each. The last batch of 10 used 37.11 mins. And fastest batch used 27.92 mins.

Reply:
:+1: